### PR TITLE
Allow multiple issuers and audiences

### PR DESCRIPTION
**Pull request by jquense**
Already upstream PR’s for this

### CODE REVIEW

File: flask_resty/jwt.py [hunk 1]
```diff
@@ -49,7 +55,8 @@ def get_request_credentials(self):
 
         try:
             payload = self.decode_token(token)
-        except InvalidTokenError:
+        except InvalidTokenError as e:
+            print(e)
```
Comment (taion) [inline flask_resty/jwt.py]: remove

**Comment by jquense** [comment]
I guess logically yeah, I stuck it here because it come up with sets, of different keys, tokens

**Comment by taion** [comment]
I'd move it upstream and merge. That or how responsive are the PyJWT devs on accepting and releasing external changes?

**Comment by jquense** [comment]
I haven't heard anything from the pyJWT folks in a week on any of my PR's so IDK. by upstream here you mean...to the JWT authentication? We can also just hold off for now and see if pyJWT merges it, its not hard to handle further down on my end until then

**Comment by taion** [comment]
That's a little annoying that they're not responding on such a small change.

If this is a blocker for you, I'd say let's move this into the parent JWT auth, and we can merge and just cut a new (non-breaking) release that uses the updated PyJWT when that's possible.

**Comment by taion** [comment]
@jquense Was this upstreamed into PyJWT?

**Comment by jquense** [comment]
no real feedback from them they on a glacial pace there

**Pull request closed without merging**